packages/scene-composer/jest.config.ts

```diff
@@ -22,6 +22,7 @@ export default merge.recursive(tsPreset, awsuiPreset, {
     '!src/augmentations/components/three-fiber/viewpoint/ViewCursorWidget.tsx', // Skipping as this is around mouse movement & is interaction based. Should be covered in end-to-end test or manual testing.
   ],
   coverageReporters: [
+    'html',
```

Comment:
Provides `HTML` output file that can be used as visual assistance for writing test coverage.